components/core/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,47 @@
+import { fireEvent, stubbedRender } from '@/testUtils';
+import SearchBar from './SearchBar';
+
+describe('<SearchBar />', () => {
+  const mockOnSubmit = jest.fn();
+  const defaultProps = { onSubmit: mockOnSubmit };
+
+  it('should render without crashing', () => {
+    const { unmount } = stubbedRender(SearchBar, {
+      props: defaultProps,
+    });
+
+    unmount();
+  });
+
+  it('renders correctly', () => {
+    stubbedRender(SearchBar, {
+      props: defaultProps,
+    });
+
+    expect(document.querySelector('.search-bar')).toMatchSnapshot();
+  });
+
+  describe('when search is initiated', () => {
+    test('the onSubmit method is called when search button is clicked', async () => {
+      stubbedRender(SearchBar, {
+        props: defaultProps,
+      });
+
+      const searchButton = document.querySelector('button[type="submit"]');
```

Comment:
Can you use testing library to find the searchButton too? So we don't click on the first button of type submit but the button where it's written submit.

And also searchButton is only used in fireEvent, we can inline that for better readability

Reply:
I went with the `data-testid` attribute here, in order to make sure we're selecting the correct elements, without effecting anything in the DOM.